packages/demo/src/demo-building-blocs/NavigationSection.tsx

```diff
@@ -11,23 +12,14 @@ export interface NavSectionProps extends NavSectionControlledProps {
 }
 
 const Section: React.FunctionComponent<NavSectionProps> = ({children, onClick, notExpandable, baseUrl, ...rest}) => {
-    const [isOpened, toggleOpened] = React.useState(true);
-    const hasMoreThanOneSection = _.size<React.ReactNode>(React.Children.map(children, _.identity)) > 1;
-
-    const handleClick = () => {
-        if (hasMoreThanOneSection) {
-            toggleOpened(!isOpened);
-        }
-        onClick?.();
-    };
-
-    return (
-        <SideNavigationMenuSection
-            {...rest}
-            expandable={!notExpandable && hasMoreThanOneSection}
-            expanded={isOpened}
-            onClick={handleClick}
-        >
+    return rest.isLink ? (
+        <Route path={baseUrl}>
+            <Link to={baseUrl}>
+                <SideNavigationMenuSection {...rest}></SideNavigationMenuSection>
+            </Link>
+        </Route>
+    ) : (
+        <SideNavigationMenuSection {...rest} expandable={false}>

```

Comment:
Why is expandable false?

Reply:
otaillon asked me to make them not expandable, so that they are expanded all the time

Reply:
I'm good with having them opened at first, but I think we should keep the functionality to close them

Reply:
I did ask him to make them non-expandable. 
The rationale is we'd like to have a content page for the section (eventually). We won't be able to have that if clicking on the section title just expand/collapse the menu.

This change should only affect the demo page, **not the admin console**

Reply:
We could still have the same behavior as the collapsible rows perhaps? Aka expand/collapse the sections only when clicking on the arrow. But still navigate to somewhere when clicking on the section header? Maybe it could be done in a later step thought.

Reply:
Why do you need the collapse functionality?

Reply:
for reasons

Reply:
I really like the solution proposed by Gael. I feel like we should stay as close as possible to what we have in the admin.

In the admin the expand/collapse state of the sections are kept in the users preferences. Meaning that someone never using a section can collapse it and never see its content if he doesn't want it

Reply:
> I feel like we should stay as close as possible to what we have in the admin.
Curious as to why you feel this way @GermainBergeron ?